images/StorageComparisonTable.md

```diff
@@ -0,0 +1,23 @@
+### Boxes vs. Other Storage Mechanisms
```

Comment:
Might as well leave it in the repo, since the demo talks about boxes. But I won't push back against a counterargument.